### DISCUSSION TITLE
Edit: Tracker issue, not this repo

### DISCUSSION THREAD

**Discussion by wwwjscom** [haugene/docker-transmission-openvpn#2503] [Q&A] [answered]
[edit: Was a tracker problem]

**Comment by pkishino** [comment, marked as answer]
Sounds like a tracker problem if it works fine with public ones

**Comment by Madseason41** [comment]
I'm having issues with PIA as well. From my laptop I can turn on the PIA VPN client and open transmission on the laptop and download a torrent (used ubuntu iso) with no problem. When I add the same torrent to the docker container nothing will download. I just get "downloading from 0 of 50 peers". Its like this for all of my torrents. Something is definitely wonky. I have been trying to fix this for two days.

**Reply by pkishino** [reply]
This sounds like a very different problem though.. please check your logs

**Reply by Madseason41** [reply]
You are correct sir. Changing this fixed it. Ugh.
WARNING: Deprecated. Found old default transmission-home folder at /data/transmission-home, setting this as TRANSMISSION_HOME. This might break in future versions.
We will fallback to this directory as long as the folder exists. Please consider moving it to /config/<transmission-home>
